docker/setup_configuration/data.yaml

```
@@ -64,9 +64,24 @@ zgw_consumers:
       user_id: open-klant
       secret: open-klant-secret
 
+    - identifier: referentielijsten-api
+      label: Referentielijsten API
+      api_root: http://referentielijsten-web.local:8000/api/v1/
+      api_type: brc
+      auth_type: zgw
+      client_id: open-klant-referentielijsten
+      user_id: open-klant-referentielijsten
+      secret: open-klant-referentielijsten-secret
```

Referentielijsten doesn't have any API authentication/authorization, so this can be left empty. The API type is technically "orc" ("overige"), but it doesn't really matter in practice afaik
```suggestion
      api_type: orc
      auth_type: no_auth
```